doc(pubsub): declare the library GA

coryan opened the pull request:
Fixes #4878 and fixes #777

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/googleapis/google-cloud-cpp/5390)
<!-- Reviewable:end -->

coryan commented:
/cc: @anguillanneuf @kamalaboulhosn FYI

devbww commented on CHANGELOG.md:
```diff
@@ -22,8 +22,8 @@
 
-While the Pub/Sub library is not GA, and breaking changes are to be expected, we
-are close enough to a GA release that we think highlighting them is important.
+While this is the first GA release of the Pub/Sub library is not GA, and
+breaking changes were to be expected, we think highlighting them is important.
```
While this is the first GA release, it is not GA??

coryan replied:
Fixed, PTAL.

kamalaboulhosn commented:
Congratulations!

devjgm reviewed (approved):
Congrats! Exciting!!!

coryan commented:
FTR, I did not merge this earlier because I did not like the performance numbers.

It turned out that I implemented the wrong throughput benchmarks.  In the process I fixed a number of small and medium sized performance problems, and some bugs with message ordering too.  Better outcome to have waited.
